gradle/scripts/travis_create_avd.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Creates and starts an emulator.
+android-update-sdk --components=sys-img-armeabi-v7a-android-22 --accept-licenses='android-sdk-license-[0-9a-f]{8}'
```

Comment:
If we use 19 as minimum sdk version, we should test with android 19 I think. Higher SDK versions generally do not remove functionality, but only add them